wavepool/views.py

```diff
@@ -14,9 +14,20 @@ def front_page(request):
             archive: the rest of the newsposts, sorted by most recent
     """
     template = loader.get_template('wavepool/frontpage.html')
-    cover_story = NewsPost.objects.all().order_by('?').first()
-    top_stories = NewsPost.objects.all().order_by('?')[:3]
-    other_stories = NewsPost.objects.all().order_by('?')
+    cover_story = None
+    top_stories = []
+    other_stories = []
+    newsposts = NewsPost.objects.all().order_by('publish_date')
+
+    i = 1
+    for n in newsposts:
+        if n.is_cover_story == True:
+            cover_story = n
+        if i < 3:
+            top_stories.append(n)
+            i = i + 1
+        else:
+            other_stories.append(n)
```

Comment:
# AC2

Cover story can get reintroduced into `top_stories` or `other_stories`


I recommend we get the news posts order by django and then slicing the list into `cover_story`, `top_stories` and `other_stories`

```
    newsposts = NewsPost.objects.order_by('-is_cover_story', '-publish_date')
    cover_story = newsposts[0]
    top_stories = newsposts[1:4]
    other_stories = newsposts[4:]
```